docs/setup/config.md

```diff
@@ -275,6 +275,10 @@ For example when running Flink on YARN on an environment with a restrictive fire
 
 - `recovery.job.delay`: (Default 'akka.ask.timeout') Defines the delay before persisted jobs are recovered in case of a recovery situation. 
 
+## Environment
+
+- `env.log.dir`: (Defaults to the `log` directory under Flink's home) Defines the directory where the Flink logs are saved.
```

Comment:
No, the user has to input an absolute path. I wouldn't recommend adding this option, a possible improvement would be to document this restriction and to check at startup for malformed paths. Would `config.sh` be a good place to perform this check?